addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiClient.java

```diff
@@ -102,8 +108,13 @@ public final Boolean isWireless() {
 
     @Override
     public String toString() {
-        return String.format("UniFiClient{mac: '%s', ip: '%s', hostname: '%s', alias: '%s', wired: %b, device: %s}",
-                mac, ip, hostname, alias, isWired(), device);
+        return String.format(
+                "UniFiClient{mac: '%s', ip: '%s', hostname: '%s', alias: '%s', wired: %b, blocked: %b, device: %s}",
+                mac, ip, hostname, alias, isWired(), blocked, device);
+    }
+
+    public void setBlocked(boolean value) {
+        this.blocked = value;
```

Comment:
See my comment above - `toString()` is usually found at the end of a class declaration.

Reply:
Changed!